pkg/runctl/cfg/pipelinerun_accessor.go

```diff
@@ -0,0 +1,59 @@
+package cfg
+
+import (
+	"github.com/SAP/stewardci-core/pkg/apis/steward/v1alpha1"
+	yaml "gopkg.in/yaml.v3"
+)
+
+// PipelineRunAccessorConfig is the config to defines a way to access a value of a pipeline run
+type PipelineRunAccessorConfig struct {
```

Comment:
Don't export this type if there is no use case outside this package.

Reply:
Unexported